### PR TITLE
Create Fact “black-eyed-susan”

### PR DESCRIPTION
**Pull request by reuterShawn**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/facts/black-eyed-susan.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url: https://www.flickr.com/photos/stanzim
+  license: CC BY-NC 2.0
+  creator: Stanley Zimny
+  alt: Black-eyed Susans
+source:
+  name: "Bees: An Identification and Native Plant Forage Guide"
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+category: habitat
+---
+During a prairie restoration, you can expect Black-eyed Susans to be one of the first flowers to bloom. Blooming from late May to late September, the Black-eyed Susan is a great pollen source for bees in the early summer. Black -eyed Susans attract long-horned, bumble, leafcutter, cuckoo, sweat and metallic green sweat bees.
```
Comment (floogulinc) [inline src/facts/black-eyed-susan.md]:
```suggestion
During a prairie restoration, you can expect Black-eyed Susans to be one of the first flowers to bloom. Blooming from late May to late September, the Black-eyed Susan is a great pollen source for bees in the early summer. Black-eyed Susans attract long-horned, bumble, leafcutter, cuckoo, sweat, and metallic green sweat bees.
```

File: src/facts/black-eyed-susan.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Black-eyed Susan
+image:
+  src: ../images/black-eyed-susan.jpg
+  url: https://www.flickr.com/photos/stanzim
+  license: CC BY-NC 2.0
+  creator: Stanley Zimny
+  alt: Black-eyed Susans
+source:
+  name: "Bees: An Identification and Native Plant Forage Guide"
```
Comment (floogulinc) [inline src/facts/black-eyed-susan.md:10]:
```suggestion
  name: Bees: An Identification and Native Plant Forage Guide
```

File: src/facts/black-eyed-susan.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Black-eyed Susan
+image:
+  src: ../images/black-eyed-susan.jpg
+  url: https://www.flickr.com/photos/stanzim
```
Comment (floogulinc) [inline src/facts/black-eyed-susan.md]: Please link to the specific image

**Review by floogulinc — CHANGES_REQUESTED** [review]

File: src/facts/black-eyed-susan.md (hunk 3 above)
Comment (reuterShawn) [reply src/facts/black-eyed-susan.md]: Sorry Paul I thought you linked it to the user's profile last time. I will get the link right next time.

**Review by floogulinc — APPROVED** [review]

**Review by kklamberty — APPROVED** [review]

**Pull request merged**